intfconfd/intfconf.cpp

```diff
@@ -0,0 +1,120 @@
+#include <string.h>
+#include "logger.h"
+#include "dbconnector.h"
+#include "producerstatetable.h"
+#include "tokenize.h"
+#include "ipprefix.h"
+#include "intfconf.h"
+#include "exec.h"
+
+using namespace std;
+using namespace swss;
+
+#define VLAN_PREFIX         "Vlan"
+#define LAG_PREFIX          "PortChannel"
+
+IntfConf::IntfConf(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, vector<string> tableNames) :
```

Comment:
>vector [](start = 81, length = 6)

Do you want to copy a vector?

Reply:
The table names for different DB are different. a vector might not be a good fit here.

Reply:
I am not sure you are talking about the same thing. I mean that use a copied instance may cost much than a reference.

Reply:
ok